vnda/utils/transform.ts

```diff
@@ -166,35 +168,93 @@ const toOffer = ({
 const toPropertyValue = (variant: VNDAProduct): PropertyValue[] =>
   Object.values(variant.properties ?? {})
     .filter(Boolean)
-    .map(({ value, name }) =>
-      value && ({
-        "@type": "PropertyValue",
-        name,
-        value,
-        valueReference: "SPECIFICATION",
-      } as PropertyValue)
-    ).filter((x): x is PropertyValue => Boolean(x));
+    .map(
+      ({ value, name }) =>
+        value &&
+        ({
+          "@type": "PropertyValue",
+          name,
+          value,
+          valueReference: "SPECIFICATION",
+        } as PropertyValue),
+    )
+    .filter((x): x is PropertyValue => Boolean(x));
 
 const toPropertyValueTags = (tags: ProductSearch["tags"]): PropertyValue[] =>
-  tags?.map((tag) =>
-    tag && ({
-      "@type": "PropertyValue",
-      name: tag.name,
-      value: JSON.stringify(tag),
-      valueReference: "TAGS",
-    } as PropertyValue)
+  tags?.map(
+    (tag) =>
+      tag &&
+      ({
+        "@type": "PropertyValue",
+        name: tag.name,
+        value: JSON.stringify(tag),
+        valueReference: "TAGS",
+      } as PropertyValue),
   );
 
 // deno-lint-ignore no-explicit-any
 const isProductVariant = (p: any): p is VariantProductSearch =>
   typeof p.id === "number";
 
-const normalizeVariants = (
-  variants: VNDAProductGroup["variants"] = [],
-): VNDAProduct[] =>
-  variants.flatMap((v) =>
-    isProductVariant(v) ? [v] : Object.values(v) as VNDAProduct[]
+// Converts a VNDAProductGroup with no variants into a VNDAProduct
+// *** If VNDAProductGroup has no variants, it is not available ***
+const buildNoVariantsProduct = (productGroup: VNDAProductGroup) => {
+  const {
+    id,
+    image_url = "",
+    installments = [],
+    name = "",
+    price = 0,
```

Comment:
Regarding the price (price) returning as 0, I have doubts if this is the most appropriate choice. I would like an explanation on why returning 0 is considered ideal.